### DISCUSSION TITLE
需要一个大模型prompt

### DISCUSSION THREAD

**Discussion by dongnanyanhai** [ofajs/discussion#11] [General]
今天想用ofa开发一个小项目，试了claude和deepseek，均不能很好理解ofa的功能和细节。

大模型辅助开发的趋势，无疑会放大新框架在知识和信息上的劣势。

对于新的开源项目来说，提供一个大模型prompt，可能是必要的。

纯粹只是建议。

**Comment by kirakiray** [comment]
感谢你的建议，其实已经有了类似的计划。ofa.js 的官网正在重构中，将会允许导出整个官网文档为 pdf 文件，可以方便拖拽到其他AI平台在提问，让AI有更好的细节索引。

**Reply by dongnanyanhai** [reply]
非常支持！

谢谢开发分享ofa这么好的项目。

**Comment by kirakiray** [comment]
现已添加skill: https://ofajs.com/cn/scenarios/ai-skill-usage
